src/test/groovy/net/anshulverma/gradle/estilo/checkstyle/checks/FileLoaderTest.groovy

```diff
@@ -50,6 +52,9 @@ class FileLoaderTest extends AbstractSpecification {
       checkType        | hash
       CheckType.GOOGLE | [44, -55, 53, -59, -34, -78, 95, -46, -92, 29, 109, -104, 126, -12, 120, -16]
       CheckType.SUN    | [123, -62, -28, 57, 21, -38, -118, -78, -54, -54, -44, 67, -21, 63, -66, -79]
+      CheckType.EMPTY  | hex2byte("3e9f1a5f8e8d821e443ead6b7daeeed6")
```

Comment:
I'm honestly unsure why you chose to represent the hashes as an array of bytes rather than just a string literal given that most tools used to hash files usually give the latter. So I offer an alternative. Let me know what you think.

Reply:
I think I was simply letting the test fail first which then prints the expected byte array and then copy pasting that array here. So, I didn't have to use another tool to get the hash.